python/rmm/rmm/_lib/__init__.py

```diff
@@ -11,5 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from rmm.pylibrmm import *
```

Comment:
*question*: Do we not still need this so that `from rmm._lib import Foo` works?

Reply:
Yes we do, adding it back